CLIENT-SPECIFICATION.md

```diff
@@ -179,6 +179,16 @@ The [`LANG` environment variable](https://www.gnu.org/software/gettext/manual/ht
 
 The [`LANGUAGE` environment variable](https://www.gnu.org/software/gettext/manual/html_node/The-LANGUAGE-variable.html) specifies a priority list of languages that a user wishes to read in. In absence of `LANG` and if this environment variable is present, a client MUST use the defined priority list to decide on the language of pages to display. If a page is not available in the user's preferred language, then a client MUST respect the user's priority list defined in the `LANGUAGE` variable, and MAY choose to notify the user that a page in their chosen language couldn't be found (perhaps along with a link to the [translations section of the contributing guide](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#translations)).
```

Comment:
With the updates below, this paragraph might need a bit of a rethink. It seems to be contradicting the below behaviour with the `LANG` and `LANGUAGE` environment variables.

(also, if `LANG` needs to be set to something other than an empty string or `C` to enable the `LANGUAGE` env  variable, what does one set it to?0

Reply:
> With the changes made below, this paragraph might need a bit of a rethink. It seems to be contradicting the below behaviour with the `LANG` and `LANGUAGE` environment variables.
> 
> (also, if `LANG` needs to be set to something other than an empty string or `C` to enable the `LANGUAGE` env variable, what does one set it to?0

LANG should be set to your primary language, for example `en_US.UTF-8` or `nl_BE.UTF-8`

Reply:
Ah, I see - thanks @principis!

Anyway, I think this paragraph needs updating to reflect the actual behaviour with respect to the GNU spec.

Reply:
```suggestion
The [`LANGUAGE` environment variable](https://www.gnu.org/software/gettext/manual/html_node/The-LANGUAGE-variable.html) specifies a priority list of languages that a user wishes to read in. The `LANG` environment variable MUST be present for `LANGUAGE` to be used, and then `LANGUAGE` takes precendence over `LANG`. If `LANG` is set to a language not in `LANGUAGE`, then `LANG` should be appended to the end of the priority list. If a page is not available in the user's preferred language, then a client MUST respect the user's priority list defined in the `LANGUAGE` variable, and MAY choose to notify the user that a page in their chosen language couldn't be found (perhaps along with a link to the [translations section of the contributing guide](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#translations)).
```

Reply:
I agree with @MasterOdin's suggestion here :+1: